aggregator/src/aggregator/aggregation_job_driver.rs

```diff
@@ -104,7 +104,7 @@ pub struct AggregationJobDriver<B> {
 
 impl<R> AggregationJobDriver<R>
 where
-    R: Backoff + Clone + Send + Sync + 'static,
+    R: BackoffBuilder + Copy + Send + Sync + 'static,
```

Comment:
nit: I think we could delete `Send + Sync` here because [`BackoffBuilder` already implies those](https://docs.rs/backon/1.5.0/backon/trait.BackoffBuilder.html).